### PR TITLE
chore: updating link to newer viz plugin blog post

### PR DESCRIPTION
**Pull request by rusackas**
### SUMMARY

Updating to the newer link, which is more relevant for the current monorepo situation

### ADDITIONAL INFORMATION

- [ ] Has associated issue:
- [ ] Required feature flags:
- [ ] Changes UI
- [ ] Includes DB Migration (follow approval process in [SIP-59](https://github.com/apache/superset/issues/13351))
  - [ ] Migration is atomic, supports rollback & is backwards-compatible
  - [ ] Confirm DB migration upgrade and downgrade tested
  - [ ] Runtime estimates and downtime expectations provided
- [ ] Introduces new feature or API
- [ ] Removes existing feature or API

### CODE REVIEW

**Comment by rusackas** [comment]
@villebro I just noticed this link... but it might be worth a look to see if there are more things worth updating in `contributing` now that we're (finally) a monorepo

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1116,7 +1116,7 @@ Then, [Updating language files](#updating-language-files).
 
 The topic of authoring new plugins, whether you'd like to contribute
 it back or not has been well documented in the
-[So, You Want to Build a Superset Viz Plugin...](https://preset.io/blog/2020-07-02-hello-world/) blog post
+[Building Custom Viz Plugins in Superset v2 (Updated for Monorepo)](https://preset.io/blog/building-custom-viz-plugins-in-superset-v2) blog post
```
Comment (villebro) [inline CONTRIBUTING.md:1119]: We now have official docs for this - should we link to the docs or maybe both? https://superset.apache.org/docs/contributing/creating-viz-plugins

Comment (rusackas) [inline CONTRIBUTING.md:1119]:
```suggestion
[the documentation](https://superset.apache.org/docs/contributing/creating-viz-plugins), or [this blog post](https://preset.io/blog/building-custom-viz-plugins-in-superset-v2).
```

Comment (rusackas) [reply CONTRIBUTING.md:1119]: @villebro let's do both!

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -1116,7 +1116,7 @@ Then, [Updating language files](#updating-language-files).
 
 The topic of authoring new plugins, whether you'd like to contribute
 it back or not has been well documented in the
-[So, You Want to Build a Superset Viz Plugin...](https://preset.io/blog/2020-07-02-hello-world/) blog post
+[the documentation](https://superset.apache.org/docs/contributing/creating-viz-plugins), or [this blog post](https://preset.io/blog/building-custom-viz-plugins-in-superset-v2).
```
Comment (rusackas) [inline CONTRIBUTING.md:1119]:
```suggestion
[the documentation](https://superset.apache.org/docs/contributing/creating-viz-plugins), and in [this blog post](https://preset.io/blog/building-custom-viz-plugins-in-superset-v2).
```

**Review by villebro — APPROVED** [review]
LGTM 🚀

**Pull request merged**